### ISSUE TITLE
Feature request: Add an option to replace the link in the "Copy link" option with an fxtwitter/nitter link

### ISSUE THREAD

**Issue by Wolfy76700** [insin/control-panel-for-twitter#522]
As title says, add an option in the Control Panel so that when the user selects "Copy Link" in the Twitter UI, the link in the clipboard is either an fxtwitter.com or nitter.net link to the same post.

**Comment by just-ero** [comment]
Looking for this as well, please!

Maybe an entirely separate button for copying the post's link could also be introduced, it would make things a lot easier in terms of sharing.

**Comment by conradsrc** [comment]
Yes, it'd be great to better customize the share button to 1) use other "fix" domains and 2) add or remove items from it, including making the button automatically copy with the desired domain instead of opening a submenu (if you disable everything else).

Personally, I just want to click the share button once and have a `fxtwitter.com` version of the link copied to my clipboard with no extra crap tacked onto the end.
